docs/xms-pageable-for-get-list-calls.md

```diff
@@ -0,0 +1,143 @@
+# XmsPageableForGetListCalls
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI(swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Get-V1-13
+
+## Output Message
+
+For all LIST APIs (a.k.a collection GETs), it is important to include the `x-ms-pageable` property.
+
+## Description
+
+`x-ms-pageable` indicates that the results of a GET call may be paginated. All LIST APIs (a.k.a collection GETs) should include this annotation.
```

Comment:
>All LIST APIs (a.k.a collection GETs) should include this annotation. [](http://example.com/codeflow?start=75&length=69)

`x-ms-pageable` indicates that the results of a GET call may be paginated. All LIST APIs (a.k.a collection GETs) should include this annotation. Specifying this extension does not mean that the service must implement paging at runtime. The service may implement this when such a requirement arises. Specifying the extension upfront future proofs the API and avoids the breaking changes that may be flagged if the extension were to be introduced in future.